### PR TITLE
[Testing Only] Start validating log rewrite

### PR DESCRIPTION
**Pull request by svanoort**
Triggers a full CI run against the log handling rewrite to get extra coverage via the durability tests

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -79,6 +79,11 @@
             <artifactId>workflow-step-api</artifactId>
             <version>2.13</version>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.marshalling</groupId>
+            <artifactId>jboss-marshalling-river</artifactId>
+            <version>1.4.12.jenkins-3</version> <!-- https://github.com/jglick/jboss-marshalling/compare/1.4.12.Final...1.4.12.jenkins-3 -->
```
Comment (jglick) [inline pom.xml]: FTR this is likely due to IDEA not grokking [this](https://github.com/jenkinsci/workflow-cps-plugin/blob/b3ab42c708d832c658c0f1d9c4935e89f2c5a147/.mvn/maven.config#L1).

**Comment by jglick** [comment]
`CpsFlowDefinition2Test.curriedClosuresInParallel` needs a minor patch, `LogActionTest.groovy` is junk and can simply be deleted, and `StepNodeTest` failures also look like they just need minor patches.

File: pom.xml [hunk 2]
```diff
@@ -143,7 +148,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-job</artifactId>
-            <version>2.21</version>
+            <version>2.26-rc847.46f833315721</version>
```
[the remark this answers is not in the document]
Comment (jglick) [reply pom.xml]: Use `2.26-rc848.9e07b6336b19` please.

**Comment by jglick** [comment]
Delete [this unused import](https://github.com/jenkinsci/workflow-cps-plugin/blob/b3ab42c708d832c658c0f1d9c4935e89f2c5a147/src/test/java/org/jenkinsci/plugins/workflow/WorkflowTest.java#L38) already!

File: pom.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (svanoort) [reply pom.xml]: Already done.

**Comment by jglick** [comment]
There is more progress in #252, FYI.

**Comment by jglick** [comment]
Now obsolete.

**Pull request closed without merging**